cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -150,15 +158,13 @@ public Boolean call() throws Exception {
 
 
   private void checkServicesWithRetry(Callable<Boolean> callable,
-                                      String exceptionMessage) throws TimeoutException {
-    int numSecs = 0;
+                                      String exceptionMessage) throws TimeoutException, InterruptedException {
```

Comment:
then you can be consistent and remove skipping interrupted exception check inside the try block.